.github/workflows/reusable-link-check.yml

```diff
@@ -13,12 +13,15 @@ jobs:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
         with:
           fetch-depth: 0 # needed for merge-base below
-
+      - uses: jdx/mise-action@bfb9fa0b029db830a8c570757cee683df207a6c5 # v2.4.0
+        with:
+          version: v2025.7.11
+          sha256: fe2d4c5c681c942b2f52bf0c71d04429ba4a5e090973514bde466a411190cd00
```

Comment:
is this needed? or is it good enough without? and by removing this would it allow us to remove the custom renovate manager?

```suggestion
```

Reply:
looks like it was an old version of the action, I've pushed an update, I like to try to limit the renovate custom managers when possible b/c they're pretty confusing